docs/advanced.md

```diff
@@ -619,3 +619,59 @@ If you do need to make HTTPS connections to a local server, for example to test
 >>> r
 Response <200 OK>
 ```
+
+## Custom Transports
+
+HTTPX's `Client` also accepts a `transport` argument. This argument allows you
+to provide a custom Transport object that will be used to perform the actual
+sending of the requests.
+
+These Transport objects must implement at least one method from
+[`httpcore`'s API](https://www.encode.io/httpcore/api/), either
+`httpcore.AsyncHTTPTransport` if you're using `AsyncClient`, or
+`httpcore.SyncHTTPTransport` if you're using `Client`.
+
+Specifically you *MUST* implement `request`. If you need additional logic for
+closing the transport you can also implement `close` or `aclose` depending on
+the type of client you're using.
```

Comment:
I think I'd probably rephrase this... "must implement at least one method from httpcore`'s API [...] Specifically you *MUST* implement `request`." scans a bit confusingly to me.

Perhaps...

"A transport instance must implement the Transport API defined by `httpcore`. You should either subclass `httpcore.AsyncHTTPTransport` to implement a transport to use with `AsyncClient`, or subclass `httpcore.SyncHTTPTransport` to implement a transport to use with `Client`."

Reply:
Yeah it reads a bit odd, your text is much clearer.